File: nattybear/level1/Amusement.java
package nattybear.level1;

class Solution {
  public long solution(int price, int money, int count) {
    long result = totalPrice(price, count) - money;
    return (result < 0) ? 0 : result;
  }

  long totalPrice(int price, int count) {
    long sum = 0;
    for (int i = 1; i <= count; i++)
      sum += price*i;
    return sum;
  }
}

public class Amusement {
  public static void main(String[] args) {
    Solution s = new Solution();
    System.out.println(s.solution(3, 20, 4));
  }
}

File: nattybear/level1/부족한-금액-계산하기/Amusement.js
function solution(price, money, count) {
  const result = total_price(price, count) - money;
  return (result < 0) ? 0 : result;
}

function total_price(price, count) {
  let sum = 0;
  for (let i = 1; i <= count; i++)
    sum += price*i;
  return sum;
}

console.log(solution(3, 20, 4));
